.appveyor.yml

```
@@ -9,6 +9,14 @@ branches:
     - /auto-backport-.*/
     - /^v\d+\.\d+\.[\dx]+-doc$/
 
+skip_commits:
+  message: /\[ci doc\]/
+  files:
+    - 'doc/**'
+    - 'galleries/examples/**'
+    - 'galleries/plot_types/**'
+    - 'galleries/tutorials/**'
```

```suggestion
    - 'galleries/**'
```
I think we can just skip the entire galleries directory.